platform/kube/client.go

```diff
@@ -111,7 +112,21 @@ func CreateRESTConfig(kubeconfig string) (config *rest.Config, err error) {
 
 // NewClient creates a client to Kubernetes API using a kubeconfig file.
 // Use an empty value for `kubeconfig` to use the in-cluster config.
+// If the kubeconfig file is empty, defaults to in-cluster config as well.
 func NewClient(kubeconfig string, km model.KindMap) (*Client, error) {
+	if kubeconfig != "" {
+		info, exists := os.Stat(kubeconfig)
+		if exists != nil {
+			return nil, fmt.Errorf("Empty kubernetes configuration file: %q", kubeconfig)
```

Comment:
I don't think this error message is quite correct since it implies the file exists but is empty. Maybe something like "kubernetes configuration file %q does not exist"instead?